functions/cloud_functions/user.js

```diff
@@ -56,4 +56,24 @@ exports.userCreated = functions.auth.user().onCreate(async user => {
         return console.log(err)
     }
 
-})
\ No newline at end of file
+})
+
+exports.userApproved = functions.firestore.document("/users/{userId}/private/info").onUpdate(async (change, context) => {
+    try {
+        await admin.firestore().doc(`newUser/${change.before.data().email}`).delete();
+    } catch (error) {
+        console.log(error.message)
+    }
+});
+
+exports.deleteUnapprovedUser = functions.firestore.document("/users/{userId}/private/info").onDelete(async (data, context) => {
+    try {
+        await admin.firestore().doc(`newUser/${data.data().email}`).delete();
```

Comment:
Delete this line... we are not doing anything to `newUser` but populating and reading it for approvals